transport/fieldset.go

```diff
@@ -0,0 +1,39 @@
+// Code generated by transport/internal/fieldset. DO NOT EDIT.
+
+package transport
+
+// FieldSet is a collection of fields grouped by their usage.
+type FieldSet struct {
+	header *Fields
+}
+
+// NewFieldSet returns an initalized FieldSet.
+func NewFieldSet() *FieldSet {
+	return &FieldSet{
+		header: &Fields{},
+	}
+}
+
+// $get retrieves the header Field by name, by performing a case-insensitive lookup.
```

Comment:
Looks like minor codegen bug here with `$get` not rendering correct name for doc strings in this file.